src/foundry/foundry.js/pointSources/soundSource.d.ts

```diff
@@ -0,0 +1,42 @@
+interface SoundSourceData extends PointSource.Data {
+  x: number;
+
+  y: number;
+
+  /** The radius of the sound effect */
+  radius: number;
+
+  walls: boolean;
+}
+
+/**
+ * A specialized subclass of the PointSource abstraction which is used to control the rendering of sound sources.
+ */
+declare class SoundSource extends PointSource {
+  /** @param object - The AmbientSound object that generates this sound source */
+  constructor(object: AmbientSound);
+
+  /** @override */
+  static sourceType: 'sound';
+
+  /**
+   * The object of data which configures how the source is rendered
+   * @defaultValue `{}`
+   */
+  data: Partial<SoundSourceData>;
+
+  /**
+   * Initialize the source with provided object data.
+   * @param data - Initial data provided to the point source
+   * @returns A reference to the initialized source
+   */
+  initialize(data?: Partial<SoundSourceData>): this;
+
+  /**
+   * Process new input data provided to the SoundSource.
+   * @param data - Initial data provided to the sound source
+   * @returns The changes compared to the prior data
+   * @internal
+   */
+  protected _initializeData(data: Partial<SoundSourceData>): void;
```

Comment:
this is documented to return the changes in foundry.js but that's incorrect, it returns `void`. Please file a bug report.